crates/iota/src/iota_commands.rs

```diff
@@ -751,8 +764,13 @@ async fn start(
     // note that this overrides the default configuration that is set when running
     // the genesis command, which sets data_ingestion_dir to None.
     #[cfg(feature = "indexer")]
-    if with_indexer.is_some() {
-        swarm_builder = swarm_builder.with_data_ingestion_dir(data_ingestion_path.clone());
+    if with_indexer.is_some() && data_ingestion_dir.is_none() {
+        data_ingestion_dir = Some(tempdir()?.into_path())
```

Comment:
Shouldn't this use the existing `data_ingestion_path`?

Reply:
Good catch, `data_ingestion_path` should be removed instead